src/core/utils/color.utils.ts

```diff
@@ -1,6 +1,6 @@
 import { CuCategoryEnum } from '../enums/cu-category.enum';
 import { CuJobEnum } from '../enums/cu-job.enum';
-import { RoleEnum } from '../enums/role.enum';
```

Comment:
Should be nice if we keep the `RoleEnum` as possible values of `RoleUserDTO` so if a role name changes we can change it easily across the app